driver-testsuite/tests/Basic/CookieTest.php

```diff
@@ -47,6 +47,13 @@ public function testCookie()
         $this->assertContains('Previous cookie: NO', $this->getSession()->getPage()->getText());
     }
 
+    public function testCookieWithSemicolon() {
```

Comment:
The opening brace of the function must be on the new line (see other methods in the file).

I recommend you setting coding standard in your IDE and that way this should be reported  or even fixed automatically.